src/ui.ts

```diff
@@ -150,10 +150,13 @@ export const DEFAULT_VIEW_STATE_BY_PRIVACY_REGIME: PrivacyRegimeToInitialViewSta
     GDPR: InitialViewState.QuickOptions,
     // Brazil
     LGPD: InitialViewState.QuickOptions,
-    // Virginia (unreachable as we don't detect this regime yet)
+    // Virginia
     CDPA: InitialViewState.NoticeAndDoNotSell,
-    // Colorado (unreachable as we don't detect this regime yet)
+    // Colorado
     CPA: InitialViewState.NoticeAndDoNotSell,
+    // Utah
+    // TODO: https://transcend.height.app/T-17644 - change Utah default view state once UCPA is in effect
```

Comment:
should we do the same for this for CPA and CDPA?

Reply:
Yeah, I'm okay with that. @bencmbrook Okay to set all 3 CPA/CDPA/UCPA all to Hidden for now?

Reply:
Oh actually CPA and CDPA are effective on Jan 1 2013 while UCPA is effective on Dec 31 2023. I think we should start enforcing CPA & CDPA early since the deadline is much closer.

Reply:
This just adds the regime right, doesn't mean we start enforcing anything? If so I'm honestly fine with adding all of them and just leaving a comment about effective date

Reply:
@anotherminh it's impossible to add the regime without also adding an enforcement preference due to types

Reply:
ya lets add whatever. once we add ip detection, we should let customers turn these regions on whenever they want to

Reply:
@eligrey right, airgap CAN start enforcing for these regimes but i'm more asking/commenting about how this PR will roll out given how things are set up today. 

Once you add these regimes + mark them as hidden by default, that means there's no regulation by default right?

Reply:
This PR will not affect enforcement; only UI experiences